charts/telemeter-reporting/templates/report-queries/machine-cpu-cores-per-cluster.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "machine-cpu-cores-per-cluster-raw"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportDataSources:
+  - "machine-cpu-cores-per-cluster"
+  columns:
```

Comment:
Needs `dt` added to the columns https://github.com/operator-framework/operator-metering/blob/master/charts/openshift-reporting/templates/report-queries/pod-cpu.yaml#L37-L38